spec/factories/miq_group.rb

```diff
@@ -1,11 +1,20 @@
 FactoryGirl.define do
+  sequence(:miq_group_description) { |n| "Test Group #{seq_padded_for_sorting(n)}" }
```

Comment:
Yea, this allows us to either generate a description or use our typical convention of `Evm-#{role}` .